_data/toc.yaml

```diff
@@ -67,9 +67,9 @@ guides:
     title: Configure GitHub Actions
 - sectiontitle: Deploy your app to the cloud
   section:
-  - path: /engine/context/aci-integration/
+  - path: /cloud/aci-integration/
     title: Docker and ACI
-  - path: /engine/context/ecs-integration/
+  - path: /cloud/ecs-integration/
```

Comment:
We're putting all these under `/cloud/`; is other content gonna be put there? Do we have a roadmap how we're gonna be unifying the docs related to this? (hard to describe; let me try); currently this is all about "compose integration with cloud". A logical location for it would be to be located under `/compose/xxx`, however, that's for the "python" compose, so might be confusing (unless we call out that it's for the "cloud" compose)

At some point, the go implementation will be equivalent in functionality, and we should probably be able to unify them; we should think of a plan how we move from "current situation" to "destination" (without moving files around too many times)